{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 436,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "from datascience import *\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 437,
   "metadata": {},
   "outputs": [],
   "source": [
    "LDLR_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/LDLR_table_refseq.txt'\n",
    "SREBF2_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/SREBF2_table_refseq.txt'\n",
    "HNF4A_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/HNF4A_table_refseq.txt'\n",
    "MYLIP_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/MYLIP_table_refseq.txt'\n",
    "ACAT2_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/ACAT2_table_refseq.txt'\n",
    "LSS_file_path = '/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/LSS_table_refseq.txt'\n",
    "pd.set_option('display.max_columns', None)\n",
    "pd.set_option('display.max_rows', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 459,
   "metadata": {},
   "outputs": [],
   "source": [
    "LDLR_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/LDLR_sequence_path.txt\"\n",
    "SREBF2_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/SREBF2_sequence_path.txt\"\n",
    "HNF4A_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/HNF4A_sequence_path.txt\"\n",
    "MYLIP_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/MYLIP_sequence_path.txt\"\n",
    "ACAT2_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/ACAT2_sequence_path.txt\"\n",
    "LSS_sequence_path = \"/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/sequence_files/LSS_sequence_path2.txt\"\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 439,
   "metadata": {},
   "outputs": [],
   "source": [
    "def sequence_table(gene, sequence_file_path, gene_file_path):\n",
    "    \"\"\"returns table with the sequence and reading frame of each exon. Inputs are exon sequence file and RefGene file from UCSC table browser.\"\"\"\n",
    "    seq_file = open(sequence_file_path).read()\n",
    "    genes = seq_file.split(\">\")\n",
    "    del genes[0]\n",
    "    gene = []\n",
    "    sequence = []\n",
    "    for i in genes:\n",
    "        gene = np.append(gene, i.split(\"=none\\n\")[0])\n",
    "        sequence = np.append(sequence, i.split(\"=none\\n\")[1].replace(\"\\n\", \"\"))\n",
    "    gene_file = pd.read_csv(gene_file_path, sep = \"\\t\").loc[0]\n",
    "    reading_frames = gene_file.loc[\"exonFrames\"].strip(\",\").split(\",\")\n",
    "    exon_starts = gene_file.loc[\"exonStarts\"].strip(\",\").split(\",\")\n",
    "    exon_ends = gene_file.loc[\"exonEnds\"].strip(\",\").split(\",\")\n",
    "    if reading_frames[0]=='-1':\n",
    "        del reading_frames[0]\n",
    "        del exon_starts[0]\n",
    "        del exon_ends[0]\n",
    "    seq_table = pd.DataFrame({\"GeneID\": gene, \"chromosome\": gene_file.loc[\"chrom\"].strip(\"chr\"), \"strand\": gene_file.loc[\"strand\"], \"sequence\": sequence, \"reading_frame\":reading_frames, \"exon_start\": exon_starts, \"exon_end\": exon_ends})\n",
    "    return seq_table"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "a = open(LSS_sequence_path).read()\n",
    "b = a.split(\">\")\n",
    "del b[0]\n",
    "c = []\n",
    "d = []\n",
    "for i in b:\n",
    "    c = np.append(c, i.split(\"=none\\n\")[0])\n",
    "    d = np.append(d, i.split(\"=none\\n\")[1].replace(\"\\n\", \"\"))\n",
    "gene_filed = pd.read_csv(LSS_file_path, sep = \"\\t\").loc[0]\n",
    "reading_framesd = gene_filed.loc[\"exonFrames\"].strip(\",\").split(\",\")\n",
    "exon_startsd = gene_filed.loc[\"exonStarts\"].strip(\",\").split(\",\")\n",
    "exon_endsd = gene_filed.loc[\"exonEnds\"].strip(\",\").split(\",\")\n",
    "if reading_framesd[0]=='-1':\n",
    "    del reading_framesd[0]\n",
    "    del exon_startsd[0]\n",
    "    del exon_endsd[0]\n",
    "\n",
    "#reading_framesd[0]=='-1'\n",
    "print(len(exon_startsd))\n",
    "print(len(exon_endsd))\n",
    "print(len(reading_framesd))\n",
    "print(len(c))\n",
    "print(len(d))\n",
    "b"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 440,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "rv_target_codons = [\"CGA\", \"TGG\"]\n",
    "fw_target_codons = [\"CGA\", \"CAA\", \"CAG\"]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "target G's in rv and C's in fw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 441,
   "metadata": {},
   "outputs": [],
   "source": [
    "def stop_gain_targets(gene, sequence_file_path, gene_file_path):\n",
    "    seq_table = sequence_table(gene, sequence_file_path, gene_file_path).drop(0)\n",
    "    rv_target_codons = [\"TGG\", \"CGA\"]\n",
    "    fw_target_codons = [\"CGA\", \"CAA\", \"CAG\"]\n",
    "    guide1s = []\n",
    "    guide2s = []\n",
    "    guide3s = []\n",
    "    guide4s = []\n",
    "    guide5s = []\n",
    "    target_strand = []\n",
    "    target_codons = []\n",
    "    target_exon = []\n",
    "    reading_frames = []\n",
    "    for i in np.arange(1, len(seq_table)):\n",
    "        row = seq_table.loc[i]\n",
    "        exon = i+1\n",
    "        reading_frame = int(row.loc[\"reading_frame\"])\n",
    "        sequence = row.loc[\"sequence\"]\n",
    "        if reading_frame == -1:\n",
    "            in_frame_sequence = sequence[2:len(sequence)]\n",
    "            adjust = 3\n",
    "        else:\n",
    "            in_frame_sequence = sequence[reading_frame:len(sequence)+1]\n",
    "            adjust = 0\n",
    "        codons = []\n",
    "        for i in np.arange(0,len(in_frame_sequence), 3):\n",
    "            codon = in_frame_sequence[i+reading_frame:(i+reading_frame+3)]\n",
    "            codons = np.append(codons, codon)\n",
    "            if codon in fw_target_codons:\n",
    "                target_base = i + int(row.loc[\"exon_start\"]) + adjust + reading_frame\n",
    "                guide_1_start = target_base+reading_frame-2\n",
    "                guide_1_end = guide_1_start+19\n",
    "                guide_2_start = guide_1_start - 1\n",
    "                guide_2_end = guide_2_start + 19\n",
    "                guide_3_start = guide_1_start - 2\n",
    "                guide_3_end = guide_3_start + 19\n",
    "                guide_4_start = guide_1_start - 3\n",
    "                guide_4_end = guide_4_start + 19\n",
    "                guide_5_start = guide_1_start - 4\n",
    "                guide_5_end = guide_5_start + 19\n",
    "                target_strand = np.append(target_strand, row.loc[\"strand\"])\n",
    "                guide1s = np.append(guide1s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_1_start)+\"-\"+str(guide_1_end))\n",
    "                guide2s = np.append(guide2s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_2_start)+\"-\"+str(guide_2_end))\n",
    "                guide3s = np.append(guide3s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_3_start)+\"-\"+str(guide_3_end))\n",
    "                guide4s = np.append(guide4s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_4_start)+\"-\"+str(guide_4_end))\n",
    "                guide5s = np.append(guide5s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_5_start)+\"-\"+str(guide_5_end))\n",
    "                target_codons = np.append(target_codons, codon)\n",
    "                target_exon = np.append(target_exon, exon)\n",
    "                reading_frames = np.append(reading_frames, reading_frame)\n",
    "            if codon in rv_target_codons:\n",
    "                target_base = i + int(row.loc[\"exon_start\"]) + adjust + reading_frame\n",
    "                guide_1_end = target_base + reading_frame + 5\n",
    "                guide_1_start = guide_1_end-19\n",
    "                guide_2_start = guide_1_start + 1\n",
    "                guide_2_end = guide_1_end + 1\n",
    "                guide_3_start = guide_1_start + 2\n",
    "                guide_3_end = guide_1_end + 2\n",
    "                guide_4_start = guide_1_start + 3\n",
    "                guide_4_end = guide_1_end + 3\n",
    "                guide_5_start = guide_1_start + 4\n",
    "                guide_5_end = guide_1_end + 4\n",
    "                target_codons = np.append(target_codons, codon)\n",
    "                target_exon = np.append(target_exon, exon)\n",
    "                reading_frames = np.append(reading_frames, reading_frame)\n",
    "                if row.loc[\"strand\"] == \"+\":\n",
    "                    target_strand = np.append(target_strand, \"-\")\n",
    "                else:\n",
    "                    target_strand = np.append(target_strand, \"+\")\n",
    "                guide1s = np.append(guide1s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_1_start)+\"-\"+str(guide_1_end))\n",
    "                guide2s = np.append(guide2s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_2_start)+\"-\"+str(guide_2_end))\n",
    "                guide3s = np.append(guide3s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_3_start)+\"-\"+str(guide_3_end))\n",
    "                guide4s = np.append(guide4s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_4_start)+\"-\"+str(guide_4_end))\n",
    "                guide5s = np.append(guide5s, \"chr\"+str(row.loc[\"chromosome\"])+\":\"+str(guide_5_start)+\"-\"+str(guide_5_end))\n",
    "            else:\n",
    "                continue\n",
    "\n",
    "    target_table = pd.DataFrame({\"gene\": gene, \"site\": \"stop/gain\", \"exon\":target_exon, \"strand\": target_strand, \"BE\": \"CBE\", \"target_codon\": target_codons, \"reading_frame\": reading_frames, \"guide1\":guide1s , \"guide2\":guide2s, \"guide3\":guide3s, \"guide4\":guide4s, \"guide5\":guide5s})\n",
    "    return target_table[target_table[\"exon\"]!=1]\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 442,
   "metadata": {},
   "outputs": [],
   "source": [
    "LDLR = stop_gain_targets(\"LDLR\", LDLR_sequence_path, LDLR_file_path)\n",
    "#w[w[\"reading_frame\"]!=0]\n",
    "#or exon !=0?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 457,
   "metadata": {},
   "outputs": [],
   "source": [
    "SREBF2 = stop_gain_targets(\"SREBF2\", SREBF2_sequence_path, SREBF2_file_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 444,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GeneID</th>\n",
       "      <th>chromosome</th>\n",
       "      <th>strand</th>\n",
       "      <th>sequence</th>\n",
       "      <th>reading_frame</th>\n",
       "      <th>exon_start</th>\n",
       "      <th>exon_end</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>hg38_refGene_NM_001287184_0 range=chr20:443906...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ATGTCGGACTGGGGCCAGGGCTTCCCCCAGGACCCACCAG</td>\n",
       "      <td>0</td>\n",
       "      <td>44390457</td>\n",
       "      <td>44390679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>hg38_refGene_NM_001287184_1 range=chr20:444060...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ACACGTCCCCATCAGAAGGCACCAACCTCAACGCGCCCAACAGCCT...</td>\n",
       "      <td>1</td>\n",
       "      <td>44406057</td>\n",
       "      <td>44406232</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>hg38_refGene_NM_001287184_2 range=chr20:444073...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ATTTAGCCGGCAGTGCGTGGTGGACAAAGACAAGAGGAACCAGTGC...</td>\n",
       "      <td>2</td>\n",
       "      <td>44407380</td>\n",
       "      <td>44407475</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>hg38_refGene_NM_001287184_3 range=chr20:444136...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>CCGTCCAGAATGAGCGGGACCGGATCAGCACTCGAAGGTCAAGCTA...</td>\n",
       "      <td>1</td>\n",
       "      <td>44413693</td>\n",
       "      <td>44413800</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>hg38_refGene_NM_001287184_4 range=chr20:444145...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ATCACCTCCCCCGTCTCCGGGATCAACGGCGACATTCGGGCGAAGA...</td>\n",
       "      <td>0</td>\n",
       "      <td>44414506</td>\n",
       "      <td>44414662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>hg38_refGene_NM_001287184_5 range=chr20:444184...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>GTGGCCCTGCTCAGAGCCCATGCTGGCGAGCACCTGCTGCTCGGAG...</td>\n",
       "      <td>0</td>\n",
       "      <td>44418424</td>\n",
       "      <td>44418512</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>hg38_refGene_NM_001287184_6 range=chr20:444197...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>GCAATGACTACATTGTCCCTCGGCACTGCCCGGAGCTGGCGGAGAT...</td>\n",
       "      <td>1</td>\n",
       "      <td>44419720</td>\n",
       "      <td>44419876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>hg38_refGene_NM_001287184_7 range=chr20:444240...</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ATGCCAAGGGGCTGAGCGATCCAGGGAAGATCAAGCGGCTGCGTTC...</td>\n",
       "      <td>1</td>\n",
       "      <td>44424017</td>\n",
       "      <td>44424643</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                              GeneID chromosome strand  \\\n",
       "0  hg38_refGene_NM_001287184_0 range=chr20:443906...         20      +   \n",
       "1  hg38_refGene_NM_001287184_1 range=chr20:444060...         20      +   \n",
       "2  hg38_refGene_NM_001287184_2 range=chr20:444073...         20      +   \n",
       "3  hg38_refGene_NM_001287184_3 range=chr20:444136...         20      +   \n",
       "4  hg38_refGene_NM_001287184_4 range=chr20:444145...         20      +   \n",
       "5  hg38_refGene_NM_001287184_5 range=chr20:444184...         20      +   \n",
       "6  hg38_refGene_NM_001287184_6 range=chr20:444197...         20      +   \n",
       "7  hg38_refGene_NM_001287184_7 range=chr20:444240...         20      +   \n",
       "\n",
       "                                            sequence reading_frame exon_start  \\\n",
       "0           ATGTCGGACTGGGGCCAGGGCTTCCCCCAGGACCCACCAG             0   44390457   \n",
       "1  ACACGTCCCCATCAGAAGGCACCAACCTCAACGCGCCCAACAGCCT...             1   44406057   \n",
       "2  ATTTAGCCGGCAGTGCGTGGTGGACAAAGACAAGAGGAACCAGTGC...             2   44407380   \n",
       "3  CCGTCCAGAATGAGCGGGACCGGATCAGCACTCGAAGGTCAAGCTA...             1   44413693   \n",
       "4  ATCACCTCCCCCGTCTCCGGGATCAACGGCGACATTCGGGCGAAGA...             0   44414506   \n",
       "5  GTGGCCCTGCTCAGAGCCCATGCTGGCGAGCACCTGCTGCTCGGAG...             0   44418424   \n",
       "6  GCAATGACTACATTGTCCCTCGGCACTGCCCGGAGCTGGCGGAGAT...             1   44419720   \n",
       "7  ATGCCAAGGGGCTGAGCGATCCAGGGAAGATCAAGCGGCTGCGTTC...             1   44424017   \n",
       "\n",
       "   exon_end  \n",
       "0  44390679  \n",
       "1  44406232  \n",
       "2  44407475  \n",
       "3  44413800  \n",
       "4  44414662  \n",
       "5  44418512  \n",
       "6  44419876  \n",
       "7  44424643  "
      ]
     },
     "execution_count": 444,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sequence_table(\"HNF4A\", HNF4A_sequence_path, HNF4A_file_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 455,
   "metadata": {},
   "outputs": [],
   "source": [
    "HNF4A = stop_gain_targets(\"HNF4A\", HNF4A_sequence_path, HNF4A_file_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 453,
   "metadata": {},
   "outputs": [],
   "source": [
    "MYLIP = stop_gain_targets(\"MYLIP\", MYLIP_sequence_path, MYLIP_file_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 454,
   "metadata": {},
   "outputs": [],
   "source": [
    "ACAT2 = stop_gain_targets(\"ACAT2\", ACAT2_sequence_path, ACAT2_file_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 461,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GeneID</th>\n",
       "      <th>chromosome</th>\n",
       "      <th>strand</th>\n",
       "      <th>sequence</th>\n",
       "      <th>reading_frame</th>\n",
       "      <th>exon_start</th>\n",
       "      <th>exon_end</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>hg38_refGene_NM_001001438_0 range=chr21:462287...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>ATGACGGAGGGCAC</td>\n",
       "      <td>0</td>\n",
       "      <td>46191065</td>\n",
       "      <td>46191235</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>hg38_refGene_NM_001001438_1 range=chr21:462284...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GTGTCTGCGGCGCCGAGGGGGCCCCTACAAGACCGAGCCCGCCACC...</td>\n",
       "      <td>2</td>\n",
       "      <td>46191880</td>\n",
       "      <td>46191959</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>hg38_refGene_NM_001001438_2 range=chr21:462275...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>AAGAATTACTTTAAGGACTTGCCCAAAGCCCACACCGCCTTTGAGG...</td>\n",
       "      <td>2</td>\n",
       "      <td>46194490</td>\n",
       "      <td>46194661</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>hg38_refGene_NM_001001438_3 range=chr21:462226...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GCCTCCTGATCACTTGCCACGTGGCACGCATCCCTCTGCCAGCCGG...</td>\n",
       "      <td>2</td>\n",
       "      <td>46195675</td>\n",
       "      <td>46195756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>hg38_refGene_NM_001001438_4 range=chr21:462218...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GCACATTGAGGATAAGTCCACCGTGTTTGGGACTGCGCTCAACTAT...</td>\n",
       "      <td>2</td>\n",
       "      <td>46196201</td>\n",
       "      <td>46196267</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>hg38_refGene_NM_001001438_5 range=chr21:462194...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GTGGTGCTGTGGCCATCCCCTCCTGGGGGAAGTTCTGGCTGGCTGT...</td>\n",
       "      <td>1</td>\n",
       "      <td>46205835</td>\n",
       "      <td>46205941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>hg38_refGene_NM_001001438_6 range=chr21:462163...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GCTGTTTCCTGACTGGGCACCGGCACACCCCTCCACACTCTGGTGC...</td>\n",
       "      <td>0</td>\n",
       "      <td>46206671</td>\n",
       "      <td>46206768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>hg38_refGene_NM_001001438_7 range=chr21:462156...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GAGCTCTATGTGGAGGACTTCGCCAGCATTGACTGGCTGGCGCAGA...</td>\n",
       "      <td>0</td>\n",
       "      <td>46207427</td>\n",
       "      <td>46207577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>hg38_refGene_NM_001001438_8 range=chr21:462151...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>CGCTCCTCAACCTGTATGAGCACCACCACAGTGCCCACCTGCGGCA...</td>\n",
       "      <td>0</td>\n",
       "      <td>46208250</td>\n",
       "      <td>46208301</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>hg38_refGene_NM_001001438_9 range=chr21:462137...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>ATCTCGAAAACCATCAACATGCTTGTGCGCTGGTATGTGGACGGGC...</td>\n",
       "      <td>0</td>\n",
       "      <td>46209553</td>\n",
       "      <td>46209625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>hg38_refGene_NM_001001438_10 range=chr21:46213...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GATGGGCCTTGACGGCATGAAAATGCAG</td>\n",
       "      <td>0</td>\n",
       "      <td>46210687</td>\n",
       "      <td>46210744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>hg38_refGene_NM_001001438_11 range=chr21:46210...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GGCACCAACGGCTCACAGATCTGGGACACCGCATTCGCCATCCAGG...</td>\n",
       "      <td>2</td>\n",
       "      <td>46213024</td>\n",
       "      <td>46213052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>hg38_refGene_NM_001001438_12 range=chr21:46209...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GCGGGCGGGCACCACAGGCCCGAGTTTTCGTCCTGCCTGCAGAAGG...</td>\n",
       "      <td>0</td>\n",
       "      <td>46213737</td>\n",
       "      <td>46213835</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>hg38_refGene_NM_001001438_13 range=chr21:46208...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GTCCCAGATAACCCTCCCGACTACCAGAAGTACTACCGCCAGATGC...</td>\n",
       "      <td>1</td>\n",
       "      <td>46215179</td>\n",
       "      <td>46215298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>hg38_refGene_NM_001001438_14 range=chr21:46207...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GGTGGCTTCTCCTTCAGTACGCTGGACTGCGGCTGGATCGTTTCTG...</td>\n",
       "      <td>0</td>\n",
       "      <td>46215684</td>\n",
       "      <td>46215793</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>hg38_refGene_NM_001001438_15 range=chr21:46206...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>CTGCTGAACATGAGAAATCCAGATGGAGGGTTCGCCACCTATGAGA...</td>\n",
       "      <td>2</td>\n",
       "      <td>46216388</td>\n",
       "      <td>46216524</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>hg38_refGene_NM_001001438_16 range=chr21:46205...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GGGACATCATGATTGACTACACCTATGTGGAGTGCACCTCAGCCGT...</td>\n",
       "      <td>1</td>\n",
       "      <td>46219475</td>\n",
       "      <td>46219572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>hg38_refGene_NM_001001438_17 range=chr21:46196...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GGAGACCCTCACGCAGGGCTTAGAGTTCTGTCGGCGGCAGCAGAGG...</td>\n",
       "      <td>2</td>\n",
       "      <td>46221853</td>\n",
       "      <td>46221975</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>hg38_refGene_NM_001001438_18 range=chr21:46195...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>CTCCTGGGGAGTTTGCTTCACCTACGGCACCTGGTTTGGCCTGGAG...</td>\n",
       "      <td>1</td>\n",
       "      <td>46222629</td>\n",
       "      <td>46222738</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>hg38_refGene_NM_001001438_19 range=chr21:46194...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GACTGCCTGTGCAGAGGTCTCCCGGGCCTGTGACTTCCTGCTGTCC...</td>\n",
       "      <td>0</td>\n",
       "      <td>46227551</td>\n",
       "      <td>46227690</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>hg38_refGene_NM_001001438_20 range=chr21:46191...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GCATCCTGACATCGAGGCCCAGGAGAGAGGAGTCCGGTGTCTACTT...</td>\n",
       "      <td>2</td>\n",
       "      <td>46228433</td>\n",
       "      <td>46228599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>hg38_refGene_NM_001001438_21 range=chr21:46191...</td>\n",
       "      <td>21</td>\n",
       "      <td>-</td>\n",
       "      <td>GAAAACATTGCTGGGGTCTTCAACAAGTCCTGTGCCATCTCCTACA...</td>\n",
       "      <td>0</td>\n",
       "      <td>46228731</td>\n",
       "      <td>46228824</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               GeneID chromosome strand  \\\n",
       "0   hg38_refGene_NM_001001438_0 range=chr21:462287...         21      -   \n",
       "1   hg38_refGene_NM_001001438_1 range=chr21:462284...         21      -   \n",
       "2   hg38_refGene_NM_001001438_2 range=chr21:462275...         21      -   \n",
       "3   hg38_refGene_NM_001001438_3 range=chr21:462226...         21      -   \n",
       "4   hg38_refGene_NM_001001438_4 range=chr21:462218...         21      -   \n",
       "5   hg38_refGene_NM_001001438_5 range=chr21:462194...         21      -   \n",
       "6   hg38_refGene_NM_001001438_6 range=chr21:462163...         21      -   \n",
       "7   hg38_refGene_NM_001001438_7 range=chr21:462156...         21      -   \n",
       "8   hg38_refGene_NM_001001438_8 range=chr21:462151...         21      -   \n",
       "9   hg38_refGene_NM_001001438_9 range=chr21:462137...         21      -   \n",
       "10  hg38_refGene_NM_001001438_10 range=chr21:46213...         21      -   \n",
       "11  hg38_refGene_NM_001001438_11 range=chr21:46210...         21      -   \n",
       "12  hg38_refGene_NM_001001438_12 range=chr21:46209...         21      -   \n",
       "13  hg38_refGene_NM_001001438_13 range=chr21:46208...         21      -   \n",
       "14  hg38_refGene_NM_001001438_14 range=chr21:46207...         21      -   \n",
       "15  hg38_refGene_NM_001001438_15 range=chr21:46206...         21      -   \n",
       "16  hg38_refGene_NM_001001438_16 range=chr21:46205...         21      -   \n",
       "17  hg38_refGene_NM_001001438_17 range=chr21:46196...         21      -   \n",
       "18  hg38_refGene_NM_001001438_18 range=chr21:46195...         21      -   \n",
       "19  hg38_refGene_NM_001001438_19 range=chr21:46194...         21      -   \n",
       "20  hg38_refGene_NM_001001438_20 range=chr21:46191...         21      -   \n",
       "21  hg38_refGene_NM_001001438_21 range=chr21:46191...         21      -   \n",
       "\n",
       "                                             sequence reading_frame  \\\n",
       "0                                      ATGACGGAGGGCAC             0   \n",
       "1   GTGTCTGCGGCGCCGAGGGGGCCCCTACAAGACCGAGCCCGCCACC...             2   \n",
       "2   AAGAATTACTTTAAGGACTTGCCCAAAGCCCACACCGCCTTTGAGG...             2   \n",
       "3   GCCTCCTGATCACTTGCCACGTGGCACGCATCCCTCTGCCAGCCGG...             2   \n",
       "4   GCACATTGAGGATAAGTCCACCGTGTTTGGGACTGCGCTCAACTAT...             2   \n",
       "5   GTGGTGCTGTGGCCATCCCCTCCTGGGGGAAGTTCTGGCTGGCTGT...             1   \n",
       "6   GCTGTTTCCTGACTGGGCACCGGCACACCCCTCCACACTCTGGTGC...             0   \n",
       "7   GAGCTCTATGTGGAGGACTTCGCCAGCATTGACTGGCTGGCGCAGA...             0   \n",
       "8   CGCTCCTCAACCTGTATGAGCACCACCACAGTGCCCACCTGCGGCA...             0   \n",
       "9   ATCTCGAAAACCATCAACATGCTTGTGCGCTGGTATGTGGACGGGC...             0   \n",
       "10                       GATGGGCCTTGACGGCATGAAAATGCAG             0   \n",
       "11  GGCACCAACGGCTCACAGATCTGGGACACCGCATTCGCCATCCAGG...             2   \n",
       "12  GCGGGCGGGCACCACAGGCCCGAGTTTTCGTCCTGCCTGCAGAAGG...             0   \n",
       "13  GTCCCAGATAACCCTCCCGACTACCAGAAGTACTACCGCCAGATGC...             1   \n",
       "14  GGTGGCTTCTCCTTCAGTACGCTGGACTGCGGCTGGATCGTTTCTG...             0   \n",
       "15  CTGCTGAACATGAGAAATCCAGATGGAGGGTTCGCCACCTATGAGA...             2   \n",
       "16  GGGACATCATGATTGACTACACCTATGTGGAGTGCACCTCAGCCGT...             1   \n",
       "17  GGAGACCCTCACGCAGGGCTTAGAGTTCTGTCGGCGGCAGCAGAGG...             2   \n",
       "18  CTCCTGGGGAGTTTGCTTCACCTACGGCACCTGGTTTGGCCTGGAG...             1   \n",
       "19  GACTGCCTGTGCAGAGGTCTCCCGGGCCTGTGACTTCCTGCTGTCC...             0   \n",
       "20  GCATCCTGACATCGAGGCCCAGGAGAGAGGAGTCCGGTGTCTACTT...             2   \n",
       "21  GAAAACATTGCTGGGGTCTTCAACAAGTCCTGTGCCATCTCCTACA...             0   \n",
       "\n",
       "   exon_start  exon_end  \n",
       "0    46191065  46191235  \n",
       "1    46191880  46191959  \n",
       "2    46194490  46194661  \n",
       "3    46195675  46195756  \n",
       "4    46196201  46196267  \n",
       "5    46205835  46205941  \n",
       "6    46206671  46206768  \n",
       "7    46207427  46207577  \n",
       "8    46208250  46208301  \n",
       "9    46209553  46209625  \n",
       "10   46210687  46210744  \n",
       "11   46213024  46213052  \n",
       "12   46213737  46213835  \n",
       "13   46215179  46215298  \n",
       "14   46215684  46215793  \n",
       "15   46216388  46216524  \n",
       "16   46219475  46219572  \n",
       "17   46221853  46221975  \n",
       "18   46222629  46222738  \n",
       "19   46227551  46227690  \n",
       "20   46228433  46228599  \n",
       "21   46228731  46228824  "
      ]
     },
     "execution_count": 461,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "LSS = stop_gain_targets(\"LSS\", LSS_sequence_path, LSS_file_path)\n",
    "sequence_table(\"LSS\", LSS_sequence_path, LSS_file_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 462,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene</th>\n",
       "      <th>site</th>\n",
       "      <th>exon</th>\n",
       "      <th>strand</th>\n",
       "      <th>BE</th>\n",
       "      <th>target_codon</th>\n",
       "      <th>reading_frame</th>\n",
       "      <th>guide1</th>\n",
       "      <th>guide2</th>\n",
       "      <th>guide3</th>\n",
       "      <th>guide4</th>\n",
       "      <th>guide5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191891-46191910</td>\n",
       "      <td>chr21:46191890-46191909</td>\n",
       "      <td>chr21:46191889-46191908</td>\n",
       "      <td>chr21:46191888-46191907</td>\n",
       "      <td>chr21:46191887-46191906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191879-46191898</td>\n",
       "      <td>chr21:46191880-46191899</td>\n",
       "      <td>chr21:46191881-46191900</td>\n",
       "      <td>chr21:46191882-46191901</td>\n",
       "      <td>chr21:46191883-46191902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191924-46191943</td>\n",
       "      <td>chr21:46191925-46191944</td>\n",
       "      <td>chr21:46191926-46191945</td>\n",
       "      <td>chr21:46191927-46191946</td>\n",
       "      <td>chr21:46191928-46191947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191939-46191958</td>\n",
       "      <td>chr21:46191938-46191957</td>\n",
       "      <td>chr21:46191937-46191956</td>\n",
       "      <td>chr21:46191936-46191955</td>\n",
       "      <td>chr21:46191935-46191954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191927-46191946</td>\n",
       "      <td>chr21:46191928-46191947</td>\n",
       "      <td>chr21:46191929-46191948</td>\n",
       "      <td>chr21:46191930-46191949</td>\n",
       "      <td>chr21:46191931-46191950</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191963-46191982</td>\n",
       "      <td>chr21:46191962-46191981</td>\n",
       "      <td>chr21:46191961-46191980</td>\n",
       "      <td>chr21:46191960-46191979</td>\n",
       "      <td>chr21:46191959-46191978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191957-46191976</td>\n",
       "      <td>chr21:46191958-46191977</td>\n",
       "      <td>chr21:46191959-46191978</td>\n",
       "      <td>chr21:46191960-46191979</td>\n",
       "      <td>chr21:46191961-46191980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46191981-46192000</td>\n",
       "      <td>chr21:46191980-46191999</td>\n",
       "      <td>chr21:46191979-46191998</td>\n",
       "      <td>chr21:46191978-46191997</td>\n",
       "      <td>chr21:46191977-46191996</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>2.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46192005-46192024</td>\n",
       "      <td>chr21:46192004-46192023</td>\n",
       "      <td>chr21:46192003-46192022</td>\n",
       "      <td>chr21:46192002-46192021</td>\n",
       "      <td>chr21:46192001-46192020</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>3.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46194549-46194568</td>\n",
       "      <td>chr21:46194550-46194569</td>\n",
       "      <td>chr21:46194551-46194570</td>\n",
       "      <td>chr21:46194552-46194571</td>\n",
       "      <td>chr21:46194553-46194572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>3.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46194624-46194643</td>\n",
       "      <td>chr21:46194623-46194642</td>\n",
       "      <td>chr21:46194622-46194641</td>\n",
       "      <td>chr21:46194621-46194640</td>\n",
       "      <td>chr21:46194620-46194639</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>4.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46195722-46195741</td>\n",
       "      <td>chr21:46195721-46195740</td>\n",
       "      <td>chr21:46195720-46195739</td>\n",
       "      <td>chr21:46195719-46195738</td>\n",
       "      <td>chr21:46195718-46195737</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>4.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46195758-46195777</td>\n",
       "      <td>chr21:46195759-46195778</td>\n",
       "      <td>chr21:46195760-46195779</td>\n",
       "      <td>chr21:46195761-46195780</td>\n",
       "      <td>chr21:46195762-46195781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>5.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46196293-46196312</td>\n",
       "      <td>chr21:46196292-46196311</td>\n",
       "      <td>chr21:46196291-46196310</td>\n",
       "      <td>chr21:46196290-46196309</td>\n",
       "      <td>chr21:46196289-46196308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>5.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46196281-46196300</td>\n",
       "      <td>chr21:46196282-46196301</td>\n",
       "      <td>chr21:46196283-46196302</td>\n",
       "      <td>chr21:46196284-46196303</td>\n",
       "      <td>chr21:46196285-46196304</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>6.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46205844-46205863</td>\n",
       "      <td>chr21:46205845-46205864</td>\n",
       "      <td>chr21:46205846-46205865</td>\n",
       "      <td>chr21:46205847-46205866</td>\n",
       "      <td>chr21:46205848-46205867</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>6.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46205856-46205875</td>\n",
       "      <td>chr21:46205857-46205876</td>\n",
       "      <td>chr21:46205858-46205877</td>\n",
       "      <td>chr21:46205859-46205878</td>\n",
       "      <td>chr21:46205860-46205879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>6.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46205883-46205902</td>\n",
       "      <td>chr21:46205884-46205903</td>\n",
       "      <td>chr21:46205885-46205904</td>\n",
       "      <td>chr21:46205886-46205905</td>\n",
       "      <td>chr21:46205887-46205906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>8.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46207446-46207465</td>\n",
       "      <td>chr21:46207447-46207466</td>\n",
       "      <td>chr21:46207448-46207467</td>\n",
       "      <td>chr21:46207449-46207468</td>\n",
       "      <td>chr21:46207450-46207469</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>8.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46207467-46207486</td>\n",
       "      <td>chr21:46207466-46207485</td>\n",
       "      <td>chr21:46207465-46207484</td>\n",
       "      <td>chr21:46207464-46207483</td>\n",
       "      <td>chr21:46207463-46207482</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>8.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46207500-46207519</td>\n",
       "      <td>chr21:46207501-46207520</td>\n",
       "      <td>chr21:46207502-46207521</td>\n",
       "      <td>chr21:46207503-46207522</td>\n",
       "      <td>chr21:46207504-46207523</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>9.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46208314-46208333</td>\n",
       "      <td>chr21:46208315-46208334</td>\n",
       "      <td>chr21:46208316-46208335</td>\n",
       "      <td>chr21:46208317-46208336</td>\n",
       "      <td>chr21:46208318-46208337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>10.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46209569-46209588</td>\n",
       "      <td>chr21:46209570-46209589</td>\n",
       "      <td>chr21:46209571-46209590</td>\n",
       "      <td>chr21:46209572-46209591</td>\n",
       "      <td>chr21:46209573-46209592</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>10.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46209614-46209633</td>\n",
       "      <td>chr21:46209613-46209632</td>\n",
       "      <td>chr21:46209612-46209631</td>\n",
       "      <td>chr21:46209611-46209630</td>\n",
       "      <td>chr21:46209610-46209629</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>13.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46213774-46213793</td>\n",
       "      <td>chr21:46213773-46213792</td>\n",
       "      <td>chr21:46213772-46213791</td>\n",
       "      <td>chr21:46213771-46213790</td>\n",
       "      <td>chr21:46213770-46213789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>13.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46213804-46213823</td>\n",
       "      <td>chr21:46213803-46213822</td>\n",
       "      <td>chr21:46213802-46213821</td>\n",
       "      <td>chr21:46213801-46213820</td>\n",
       "      <td>chr21:46213800-46213819</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>14.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46215194-46215213</td>\n",
       "      <td>chr21:46215193-46215212</td>\n",
       "      <td>chr21:46215192-46215211</td>\n",
       "      <td>chr21:46215191-46215210</td>\n",
       "      <td>chr21:46215190-46215209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>14.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46215182-46215201</td>\n",
       "      <td>chr21:46215183-46215202</td>\n",
       "      <td>chr21:46215184-46215203</td>\n",
       "      <td>chr21:46215185-46215204</td>\n",
       "      <td>chr21:46215186-46215205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>14.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAA</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46215224-46215243</td>\n",
       "      <td>chr21:46215223-46215242</td>\n",
       "      <td>chr21:46215222-46215241</td>\n",
       "      <td>chr21:46215221-46215240</td>\n",
       "      <td>chr21:46215220-46215239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>15.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46215703-46215722</td>\n",
       "      <td>chr21:46215704-46215723</td>\n",
       "      <td>chr21:46215705-46215724</td>\n",
       "      <td>chr21:46215706-46215725</td>\n",
       "      <td>chr21:46215707-46215726</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>15.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46215766-46215785</td>\n",
       "      <td>chr21:46215765-46215784</td>\n",
       "      <td>chr21:46215764-46215783</td>\n",
       "      <td>chr21:46215763-46215782</td>\n",
       "      <td>chr21:46215762-46215781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>16.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46216405-46216424</td>\n",
       "      <td>chr21:46216404-46216423</td>\n",
       "      <td>chr21:46216403-46216422</td>\n",
       "      <td>chr21:46216402-46216421</td>\n",
       "      <td>chr21:46216401-46216420</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>16.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46216441-46216460</td>\n",
       "      <td>chr21:46216442-46216461</td>\n",
       "      <td>chr21:46216443-46216462</td>\n",
       "      <td>chr21:46216444-46216463</td>\n",
       "      <td>chr21:46216445-46216464</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>17.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46219523-46219542</td>\n",
       "      <td>chr21:46219522-46219541</td>\n",
       "      <td>chr21:46219521-46219540</td>\n",
       "      <td>chr21:46219520-46219539</td>\n",
       "      <td>chr21:46219519-46219538</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>18.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46221864-46221883</td>\n",
       "      <td>chr21:46221863-46221882</td>\n",
       "      <td>chr21:46221862-46221881</td>\n",
       "      <td>chr21:46221861-46221880</td>\n",
       "      <td>chr21:46221860-46221879</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>18.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46221888-46221907</td>\n",
       "      <td>chr21:46221887-46221906</td>\n",
       "      <td>chr21:46221886-46221905</td>\n",
       "      <td>chr21:46221885-46221904</td>\n",
       "      <td>chr21:46221884-46221903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>18.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46221891-46221910</td>\n",
       "      <td>chr21:46221890-46221909</td>\n",
       "      <td>chr21:46221889-46221908</td>\n",
       "      <td>chr21:46221888-46221907</td>\n",
       "      <td>chr21:46221887-46221906</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>18.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46221897-46221916</td>\n",
       "      <td>chr21:46221898-46221917</td>\n",
       "      <td>chr21:46221899-46221918</td>\n",
       "      <td>chr21:46221900-46221919</td>\n",
       "      <td>chr21:46221901-46221920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>19.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46222656-46222675</td>\n",
       "      <td>chr21:46222657-46222676</td>\n",
       "      <td>chr21:46222658-46222677</td>\n",
       "      <td>chr21:46222659-46222678</td>\n",
       "      <td>chr21:46222660-46222679</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>19.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>1.0</td>\n",
       "      <td>chr21:46222674-46222693</td>\n",
       "      <td>chr21:46222675-46222694</td>\n",
       "      <td>chr21:46222676-46222695</td>\n",
       "      <td>chr21:46222677-46222696</td>\n",
       "      <td>chr21:46222678-46222697</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>20.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46227639-46227658</td>\n",
       "      <td>chr21:46227638-46227657</td>\n",
       "      <td>chr21:46227637-46227656</td>\n",
       "      <td>chr21:46227636-46227655</td>\n",
       "      <td>chr21:46227635-46227654</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>20.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CGA</td>\n",
       "      <td>0.0</td>\n",
       "      <td>chr21:46227627-46227646</td>\n",
       "      <td>chr21:46227628-46227647</td>\n",
       "      <td>chr21:46227629-46227648</td>\n",
       "      <td>chr21:46227630-46227649</td>\n",
       "      <td>chr21:46227631-46227650</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>21.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46228450-46228469</td>\n",
       "      <td>chr21:46228449-46228468</td>\n",
       "      <td>chr21:46228448-46228467</td>\n",
       "      <td>chr21:46228447-46228466</td>\n",
       "      <td>chr21:46228446-46228465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>21.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46228483-46228502</td>\n",
       "      <td>chr21:46228482-46228501</td>\n",
       "      <td>chr21:46228481-46228500</td>\n",
       "      <td>chr21:46228480-46228499</td>\n",
       "      <td>chr21:46228479-46228498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>21.0</td>\n",
       "      <td>+</td>\n",
       "      <td>CBE</td>\n",
       "      <td>TGG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46228489-46228508</td>\n",
       "      <td>chr21:46228490-46228509</td>\n",
       "      <td>chr21:46228491-46228510</td>\n",
       "      <td>chr21:46228492-46228511</td>\n",
       "      <td>chr21:46228493-46228512</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>LSS</td>\n",
       "      <td>stop/gain</td>\n",
       "      <td>21.0</td>\n",
       "      <td>-</td>\n",
       "      <td>CBE</td>\n",
       "      <td>CAG</td>\n",
       "      <td>2.0</td>\n",
       "      <td>chr21:46228507-46228526</td>\n",
       "      <td>chr21:46228506-46228525</td>\n",
       "      <td>chr21:46228505-46228524</td>\n",
       "      <td>chr21:46228504-46228523</td>\n",
       "      <td>chr21:46228503-46228522</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   gene       site  exon strand   BE target_codon  reading_frame  \\\n",
       "0   LSS  stop/gain   2.0      -  CBE          CGA            2.0   \n",
       "1   LSS  stop/gain   2.0      +  CBE          CGA            2.0   \n",
       "2   LSS  stop/gain   2.0      +  CBE          TGG            2.0   \n",
       "3   LSS  stop/gain   2.0      -  CBE          CGA            2.0   \n",
       "4   LSS  stop/gain   2.0      +  CBE          CGA            2.0   \n",
       "5   LSS  stop/gain   2.0      -  CBE          CAG            2.0   \n",
       "6   LSS  stop/gain   2.0      +  CBE          TGG            2.0   \n",
       "7   LSS  stop/gain   2.0      -  CBE          CAG            2.0   \n",
       "8   LSS  stop/gain   2.0      -  CBE          CAG            2.0   \n",
       "9   LSS  stop/gain   3.0      +  CBE          TGG            2.0   \n",
       "10  LSS  stop/gain   3.0      -  CBE          CAG            2.0   \n",
       "11  LSS  stop/gain   4.0      -  CBE          CAG            2.0   \n",
       "12  LSS  stop/gain   4.0      +  CBE          TGG            2.0   \n",
       "13  LSS  stop/gain   5.0      -  CBE          CGA            2.0   \n",
       "14  LSS  stop/gain   5.0      +  CBE          CGA            2.0   \n",
       "15  LSS  stop/gain   6.0      +  CBE          TGG            1.0   \n",
       "16  LSS  stop/gain   6.0      +  CBE          TGG            1.0   \n",
       "17  LSS  stop/gain   6.0      +  CBE          TGG            1.0   \n",
       "18  LSS  stop/gain   8.0      +  CBE          TGG            0.0   \n",
       "19  LSS  stop/gain   8.0      -  CBE          CAG            0.0   \n",
       "20  LSS  stop/gain   8.0      +  CBE          TGG            0.0   \n",
       "21  LSS  stop/gain   9.0      +  CBE          TGG            0.0   \n",
       "22  LSS  stop/gain  10.0      +  CBE          TGG            0.0   \n",
       "23  LSS  stop/gain  10.0      -  CBE          CAG            0.0   \n",
       "24  LSS  stop/gain  13.0      -  CBE          CAG            0.0   \n",
       "25  LSS  stop/gain  13.0      -  CBE          CAG            0.0   \n",
       "26  LSS  stop/gain  14.0      -  CBE          CGA            1.0   \n",
       "27  LSS  stop/gain  14.0      +  CBE          CGA            1.0   \n",
       "28  LSS  stop/gain  14.0      -  CBE          CAA            1.0   \n",
       "29  LSS  stop/gain  15.0      +  CBE          TGG            0.0   \n",
       "30  LSS  stop/gain  15.0      -  CBE          CAG            0.0   \n",
       "31  LSS  stop/gain  16.0      -  CBE          CAG            2.0   \n",
       "32  LSS  stop/gain  16.0      +  CBE          TGG            2.0   \n",
       "33  LSS  stop/gain  17.0      -  CBE          CAG            1.0   \n",
       "34  LSS  stop/gain  18.0      -  CBE          CAG            2.0   \n",
       "35  LSS  stop/gain  18.0      -  CBE          CAG            2.0   \n",
       "36  LSS  stop/gain  18.0      -  CBE          CAG            2.0   \n",
       "37  LSS  stop/gain  18.0      +  CBE          TGG            2.0   \n",
       "38  LSS  stop/gain  19.0      +  CBE          TGG            1.0   \n",
       "39  LSS  stop/gain  19.0      +  CBE          TGG            1.0   \n",
       "40  LSS  stop/gain  20.0      -  CBE          CGA            0.0   \n",
       "41  LSS  stop/gain  20.0      +  CBE          CGA            0.0   \n",
       "42  LSS  stop/gain  21.0      -  CBE          CAG            2.0   \n",
       "43  LSS  stop/gain  21.0      -  CBE          CAG            2.0   \n",
       "44  LSS  stop/gain  21.0      +  CBE          TGG            2.0   \n",
       "45  LSS  stop/gain  21.0      -  CBE          CAG            2.0   \n",
       "\n",
       "                     guide1                   guide2                   guide3  \\\n",
       "0   chr21:46191891-46191910  chr21:46191890-46191909  chr21:46191889-46191908   \n",
       "1   chr21:46191879-46191898  chr21:46191880-46191899  chr21:46191881-46191900   \n",
       "2   chr21:46191924-46191943  chr21:46191925-46191944  chr21:46191926-46191945   \n",
       "3   chr21:46191939-46191958  chr21:46191938-46191957  chr21:46191937-46191956   \n",
       "4   chr21:46191927-46191946  chr21:46191928-46191947  chr21:46191929-46191948   \n",
       "5   chr21:46191963-46191982  chr21:46191962-46191981  chr21:46191961-46191980   \n",
       "6   chr21:46191957-46191976  chr21:46191958-46191977  chr21:46191959-46191978   \n",
       "7   chr21:46191981-46192000  chr21:46191980-46191999  chr21:46191979-46191998   \n",
       "8   chr21:46192005-46192024  chr21:46192004-46192023  chr21:46192003-46192022   \n",
       "9   chr21:46194549-46194568  chr21:46194550-46194569  chr21:46194551-46194570   \n",
       "10  chr21:46194624-46194643  chr21:46194623-46194642  chr21:46194622-46194641   \n",
       "11  chr21:46195722-46195741  chr21:46195721-46195740  chr21:46195720-46195739   \n",
       "12  chr21:46195758-46195777  chr21:46195759-46195778  chr21:46195760-46195779   \n",
       "13  chr21:46196293-46196312  chr21:46196292-46196311  chr21:46196291-46196310   \n",
       "14  chr21:46196281-46196300  chr21:46196282-46196301  chr21:46196283-46196302   \n",
       "15  chr21:46205844-46205863  chr21:46205845-46205864  chr21:46205846-46205865   \n",
       "16  chr21:46205856-46205875  chr21:46205857-46205876  chr21:46205858-46205877   \n",
       "17  chr21:46205883-46205902  chr21:46205884-46205903  chr21:46205885-46205904   \n",
       "18  chr21:46207446-46207465  chr21:46207447-46207466  chr21:46207448-46207467   \n",
       "19  chr21:46207467-46207486  chr21:46207466-46207485  chr21:46207465-46207484   \n",
       "20  chr21:46207500-46207519  chr21:46207501-46207520  chr21:46207502-46207521   \n",
       "21  chr21:46208314-46208333  chr21:46208315-46208334  chr21:46208316-46208335   \n",
       "22  chr21:46209569-46209588  chr21:46209570-46209589  chr21:46209571-46209590   \n",
       "23  chr21:46209614-46209633  chr21:46209613-46209632  chr21:46209612-46209631   \n",
       "24  chr21:46213774-46213793  chr21:46213773-46213792  chr21:46213772-46213791   \n",
       "25  chr21:46213804-46213823  chr21:46213803-46213822  chr21:46213802-46213821   \n",
       "26  chr21:46215194-46215213  chr21:46215193-46215212  chr21:46215192-46215211   \n",
       "27  chr21:46215182-46215201  chr21:46215183-46215202  chr21:46215184-46215203   \n",
       "28  chr21:46215224-46215243  chr21:46215223-46215242  chr21:46215222-46215241   \n",
       "29  chr21:46215703-46215722  chr21:46215704-46215723  chr21:46215705-46215724   \n",
       "30  chr21:46215766-46215785  chr21:46215765-46215784  chr21:46215764-46215783   \n",
       "31  chr21:46216405-46216424  chr21:46216404-46216423  chr21:46216403-46216422   \n",
       "32  chr21:46216441-46216460  chr21:46216442-46216461  chr21:46216443-46216462   \n",
       "33  chr21:46219523-46219542  chr21:46219522-46219541  chr21:46219521-46219540   \n",
       "34  chr21:46221864-46221883  chr21:46221863-46221882  chr21:46221862-46221881   \n",
       "35  chr21:46221888-46221907  chr21:46221887-46221906  chr21:46221886-46221905   \n",
       "36  chr21:46221891-46221910  chr21:46221890-46221909  chr21:46221889-46221908   \n",
       "37  chr21:46221897-46221916  chr21:46221898-46221917  chr21:46221899-46221918   \n",
       "38  chr21:46222656-46222675  chr21:46222657-46222676  chr21:46222658-46222677   \n",
       "39  chr21:46222674-46222693  chr21:46222675-46222694  chr21:46222676-46222695   \n",
       "40  chr21:46227639-46227658  chr21:46227638-46227657  chr21:46227637-46227656   \n",
       "41  chr21:46227627-46227646  chr21:46227628-46227647  chr21:46227629-46227648   \n",
       "42  chr21:46228450-46228469  chr21:46228449-46228468  chr21:46228448-46228467   \n",
       "43  chr21:46228483-46228502  chr21:46228482-46228501  chr21:46228481-46228500   \n",
       "44  chr21:46228489-46228508  chr21:46228490-46228509  chr21:46228491-46228510   \n",
       "45  chr21:46228507-46228526  chr21:46228506-46228525  chr21:46228505-46228524   \n",
       "\n",
       "                     guide4                   guide5  \n",
       "0   chr21:46191888-46191907  chr21:46191887-46191906  \n",
       "1   chr21:46191882-46191901  chr21:46191883-46191902  \n",
       "2   chr21:46191927-46191946  chr21:46191928-46191947  \n",
       "3   chr21:46191936-46191955  chr21:46191935-46191954  \n",
       "4   chr21:46191930-46191949  chr21:46191931-46191950  \n",
       "5   chr21:46191960-46191979  chr21:46191959-46191978  \n",
       "6   chr21:46191960-46191979  chr21:46191961-46191980  \n",
       "7   chr21:46191978-46191997  chr21:46191977-46191996  \n",
       "8   chr21:46192002-46192021  chr21:46192001-46192020  \n",
       "9   chr21:46194552-46194571  chr21:46194553-46194572  \n",
       "10  chr21:46194621-46194640  chr21:46194620-46194639  \n",
       "11  chr21:46195719-46195738  chr21:46195718-46195737  \n",
       "12  chr21:46195761-46195780  chr21:46195762-46195781  \n",
       "13  chr21:46196290-46196309  chr21:46196289-46196308  \n",
       "14  chr21:46196284-46196303  chr21:46196285-46196304  \n",
       "15  chr21:46205847-46205866  chr21:46205848-46205867  \n",
       "16  chr21:46205859-46205878  chr21:46205860-46205879  \n",
       "17  chr21:46205886-46205905  chr21:46205887-46205906  \n",
       "18  chr21:46207449-46207468  chr21:46207450-46207469  \n",
       "19  chr21:46207464-46207483  chr21:46207463-46207482  \n",
       "20  chr21:46207503-46207522  chr21:46207504-46207523  \n",
       "21  chr21:46208317-46208336  chr21:46208318-46208337  \n",
       "22  chr21:46209572-46209591  chr21:46209573-46209592  \n",
       "23  chr21:46209611-46209630  chr21:46209610-46209629  \n",
       "24  chr21:46213771-46213790  chr21:46213770-46213789  \n",
       "25  chr21:46213801-46213820  chr21:46213800-46213819  \n",
       "26  chr21:46215191-46215210  chr21:46215190-46215209  \n",
       "27  chr21:46215185-46215204  chr21:46215186-46215205  \n",
       "28  chr21:46215221-46215240  chr21:46215220-46215239  \n",
       "29  chr21:46215706-46215725  chr21:46215707-46215726  \n",
       "30  chr21:46215763-46215782  chr21:46215762-46215781  \n",
       "31  chr21:46216402-46216421  chr21:46216401-46216420  \n",
       "32  chr21:46216444-46216463  chr21:46216445-46216464  \n",
       "33  chr21:46219520-46219539  chr21:46219519-46219538  \n",
       "34  chr21:46221861-46221880  chr21:46221860-46221879  \n",
       "35  chr21:46221885-46221904  chr21:46221884-46221903  \n",
       "36  chr21:46221888-46221907  chr21:46221887-46221906  \n",
       "37  chr21:46221900-46221919  chr21:46221901-46221920  \n",
       "38  chr21:46222659-46222678  chr21:46222660-46222679  \n",
       "39  chr21:46222677-46222696  chr21:46222678-46222697  \n",
       "40  chr21:46227636-46227655  chr21:46227635-46227654  \n",
       "41  chr21:46227630-46227649  chr21:46227631-46227650  \n",
       "42  chr21:46228447-46228466  chr21:46228446-46228465  \n",
       "43  chr21:46228480-46228499  chr21:46228479-46228498  \n",
       "44  chr21:46228492-46228511  chr21:46228493-46228512  \n",
       "45  chr21:46228504-46228523  chr21:46228503-46228522  "
      ]
     },
     "execution_count": 462,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "LSS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 458,
   "metadata": {},
   "outputs": [],
   "source": [
    "#LDLR.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/LDLR_CBE_controls.csv')\n",
    "#SREBF2.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/SEBF2_CBE_controls.csv')\n",
    "#HNF4A.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/HNF4A_CBE_controls.csv')\n",
    "#MYLIP.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/MYLIP_CBE_controls.csv')\n",
    "#ACAT2.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/ACAT2_CBE_controls.csv')\n",
    "LSS.to_csv('/Users/allisonjames/Documents/Rotations/Sherwood Lab/BE screens/LSS_CBE_controls.csv')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
